.github/workflows/org-gate.yml

```
@@ -1,151 +1,153 @@
-        env:
-          GH_TOKEN: ${{ github.token }}
-          CR_API: "repos/${{ github.repository }}/commits/${{ github.event.pull_request.head.sha }}/check-runs"   # zizmor：模板值经 env 间接引用
-        run: |
-          LOCAL=$(gh api "$CR_API" \
-            --jq '[.check_runs[] | select(.name == "gate") | .conclusion] | first // empty' 2>/dev/null || true)
-          if [[ -n "$LOCAL" && "$LOCAL" != "success" ]]; then
-            echo "::warning::双轨分叉（T3 观察期数据）：本地 gate=$LOCAL 而 org-gate=success——须归因后才能退役旧轨（ADR-0046 决策 3）"
-          elif [[ -n "$LOCAL" ]]; then
-            echo "双轨一致：本地 gate=$LOCAL，org-gate=success"
-          fi
+name: org-gate
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)